docker/harbor/harbor.yml

```
@@ -0,0 +1,17 @@
+database:
+  type: postgresql
+  ssl_mode: disable
+  host: postgres-main
+  port: 5432
+  username: cfc_admin
+  password: cfc_password_dev
+  dbname: harbor
+database:
```

YAMLlint (1.37.1)

[error] duplication of key "database" in mapping

(key-duplicates)